karma.conf.js

```diff
@@ -16,6 +16,7 @@ module.exports = function (config) {
             'karma-phantomjs-launcher',
             'karma-coverage'
         ],
+        usePolling: true,
```

Comment:
Will remove - I was looking into this as a potential fix for the hosted build agent in VSO.